client/pages/old-jokes.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export default class OldJokes extends React.Component {
+  constructor(props) {
+    super(props);
+    this.super = {
+      oldJoke: ''
+    };
+  }
+
+  render() {
+    return (
+    <div></div>
```

Comment:
It helps to throw at least an `h1` in your output so you can test all of your routing and stuff to visually see if it is working or not.

```JSX
return (
  <div>
    <h1>Old Joke Page</h1>
  </div>
);
```